sql/modules/Reconciliation.sql

```diff
@@ -44,6 +50,18 @@ $$Submits a reconciliation report for approval.
 in_line_ids is used to specify which report lines are cleared, finalizing the
 report.$$;
 
+CREATE OR REPLACE FUNCTION reconciliation__check_balanced(in_report_id int
+      ) RETURNS NUMERIC AS
+$$
+    SELECT COALESCE((
+          SELECT SUM(crl.our_balance) - cr.their_total
+            FROM cr_report_line crl
+            JOIN cr_report cr ON cr.id = crl.report_id
```

Comment:
if there is a report with no lines, this causes the function to indicate "balanced" (sum of zero lines is NULL, coalesced to zero). Is that the correct behaviour?

Reply:
That was the intent. In the application, we want only the amount, in tests we want 0 and it is the actual of an empty ledger or report.